frontend/src/host_orchestrator/main.go

```diff
@@ -57,6 +56,9 @@ func startHttpServer(port string) error {
 }
 
 func startHttpsServer(port string, certPath string, keyPath string) error {
+	if port == "" {
```

Comment:
When reading the code I think having a more upfront logic towards starting the https server is easier to follow, and let the function panic if `port` was empty.

```
starters := []func() error{
		func() error { return operator.SetupDeviceEndpoint(pool, config, socketPath)() },
		func() error { return startHttpServer(httpPort) },
}
if httpsPort != "" {
  starters = append(starters, func() error { return startHttpsServer(httpsPort, certPath, keyPath) })
}
```

Reply:
Done